package.json

```diff
@@ -12,11 +12,26 @@
     "assets": "sh scripts/assets.sh",
     "deploy": "sh scripts/deploy.sh",
     "mocha": "sh scripts/mocha.sh",
+    "precommit": "lint-staged",
     "prepare": "patch-package",
+    "prettier-write": "yarn run prettier --write",
     "start": "sh scripts/start.sh",
     "test": "sh scripts/test.sh",
     "webpack": "webpack"
   },
+  "prettier": {
+    "arrowParens": "always",
+    "bracketSpacing": true,
+    "semi": false,
+    "singleQuote": true,
```

Comment:
Sorry y'all 😄 we type less JSON than JS and our wrist muscles will thank us later